server/app/Icepeak/Main.hs

```diff
@@ -123,7 +127,9 @@ logRunCoreResult logger rcr = do
   where
     handleLog name exc
       | Just (_ :: AsyncException) <- fromException exc = pure ()
-      | otherwise = Logger.postLog logger $ name <> " stopped with an exception: " <> Text.pack (show exc)
+      | otherwise = do
+            Logger.postLog logger $ name <> " stopped with an exception: " <> Text.pack (show exc)
+            SentryLogging.logException "Log Run Core Error" mSentryService exc
```

Comment:
The error that is logged to sentry should also contain the `name` of the thread that crashed.